resources/aws/subnet.go

```diff
@@ -145,7 +145,7 @@ func (s Subnet) GetID() (string, error) {
 	return *subnet.SubnetId, nil
 }
 
-func (s *Subnet) MakePublic(routeTable *RouteTable) error {
+func (s *Subnet) AssociateRouteTable(routeTable *RouteTable) error {
```

Comment:
Previously this was part of the subnet MakePublic method. But we also need to associate the private route table so it's now split out.